hack/lib/util.sh

```diff
@@ -513,6 +513,14 @@ kube::util::ensure_godep_version() {
   godep version
 }
 
+# Checks that the GOPATH is simple, i.e. consists only of one directory, not multiple.
+kube::util::ensure_single_dir_gopath() {
```

Comment:
I like this 👍 ! Would be useful to add on to the staging godeps dockerized script I've got too.